webapp/src/App.vue

```diff
@@ -54,35 +60,52 @@ export default {
 
   methods: {
     // creation of new item
-    createItem: function(newTitle) {
+    createItem(newTitle) {
       this.news.push({
           id: this.counter++,
           title: newTitle,
           votes: 0
       });
-      this.sortItems();
+      // this.sortItems();
     },
     // deleting an item
-    deleteItem: function(item) {
+    deleteItem(item) {
       this.news = this.news.filter(el => el.id != item.id);
     },
     //updating an item
-    updateItem: function(item) {
+    updateItem(item) {
       this.news.map(el => {
         if (el.id == item.id) {
           return item
         }
         return el
       })
-      this.sortItems();
+      // this.sortItems();
     },
     //sorting of items 
-    sortItems: function() {
-      this.news = [...this.news].sort((a,b) => (a.votes < b.votes) ? 1 : ((b.votes < a.votes) ? -1 : 0));
+    sortItems(sort="ascending") {
+      this.sortOrder = sort;
+      // if (sort=="ascending"){
+      // this.news = [...this.news].sort((a,b) => (a.votes < b.votes) ? 1 : ((b.votes < a.votes) ? -1 : 0));
+      
+      // }
+      // else{
+      //   this.news = [...this.news].sort((a,b) => (a.votes > b.votes) ? 1 : ((b.votes > a.votes) ? -1 : 0));
+      // }
```

Comment:
Do you still need that?
If this is unnecessary, it's better to take care of code before it starts to rot.